india_forecast_app/adjuster.py

```diff
@@ -0,0 +1,91 @@
+""" Adjuster code, adjust forecast by last 7 days of ME"""
+from datetime import datetime, timedelta
+from typing import Optional
+
+import pandas as pd
+from pvsite_datamodel.sqlmodels import ForecastSQL, ForecastValueSQL, GenerationSQL
+from sqlalchemy import INT, cast, text
+from sqlalchemy.sql import func
+
+# Wad to get all the forecast for the last 7 days made, at this time.
+# And find the ME for each forecast horizon
+
+"""
+Here is the SQL query that it based off
```

Comment:
Great that you have added this in, much easier to read then the sql alchemy bit below